corehq/apps/userreports/reports/view.py

```diff
@@ -104,7 +104,7 @@ def query_dict_to_dict(query_dict, domain):
     :param domain:
     :return: a dict
     """
-    request_dict = json_request(query_dict)
+    request_dict = json_request(query_dict, booleans_as_strings=True)
```

Comment:
I changed this in favour of https://github.com/dimagi/commcare-hq/pull/16575/ (that does more explicit casting and has tests)